docs/docs/guides/How to Add Digest to In-App Notifications.md

```diff
@@ -0,0 +1,293 @@
+
+Note: Note: Each node that is added below the digest node will only be triggered after the specified time interval
+
+For example, in our case, say we want the In-App notification to be sent after every 6 hours. Then we’ll add the ‘digest’ action below the ‘trigger node’ and the ‘in-app’ channel node below the ‘digest’ node as shown below:
+
+![Add digest action below the trigger node and in-app channel node below the digest node](https://res.cloudinary.com/dxc6bnman/image/upload/v1688131735/guides/Screenshot_2023-06-25_at_11.01.23_PM_vsvrjh.png)
+
+The time interval after which you want the notifications to be sent can be configured by selecting the ‘digest’ node:
+
+![Configure the time interval in the digest node](https://res.cloudinary.com/dxc6bnman/image/upload/v1688131926/guides/Screenshot_2023-06-25_at_11.02.17_PM_vsmjg9.png)
+
+Once, you’ve configured it. Go ahead and configure the ‘in-app’ channel as per your need:
+
+![Various options in the in-app channel](https://res.cloudinary.com/dxc6bnman/image/upload/v1688132220/guides/SCR-20230630-qqbi_nqftzz.png)
+
+Here’s a brief explanation of all the options:
+
+- **1-Preview**: This shows you a glimpse of what each notification item will look like in the Notification Center UI.
+- **2-Avatar:** If turned on, each notification item will show the avatar of the subscriber.
+- **3-Action:** With this, you can add a primary and secondary call to action button to each notification item.
+- **4-Notification Feeds:** This displays a stream of specific notifications. You can have multiple feeds to show specific notifications in multiple tabs.
+- **5-Redirect URL** - This is the URL to which a subscriber can be directed when they click on a notification item.
+- **6-Filter** - This feature allows you to configure the criteria for delivering notifications. For instance, you can apply a filter based on a subscriber's online status to send them an email if they were online within the last hour. Read [more about filters](https://docs.novu.co/platform/step-filter/#subscriber-seen--read-filters).
+
+In our case, we’re going to use the following:
+
+![handlebar code that we're going to use](https://res.cloudinary.com/dxc6bnman/image/upload/v1688132428/guides/SCR-20230630-qsjg_xl4kk9.png)
+
+Once you’re done configuring this to your liking, click on the ‘update’ button on the top right. You can read more about it [here](https://docs.novu.co/platform/digest/#writing-digest-templates)
+
+It’ll automatically create a trigger code that you can use in your app. To get it, click on the ‘get snippet’ button on the top right and copy it:
+
+![code snippet to trigger ](https://res.cloudinary.com/dxc6bnman/image/upload/v1688132689/guides/Screenshot_2023-06-25_at_11.19.29_PM_wruvxs.png)
+
+Now, let’s see how to add Novu to our app!
+
+## Adding Novu to the Backend and Configure it
```

Comment:
I made a comment about this, still no change.

Please change "Adding" to "Add"